packages/melody-types/src/index.js

```diff
@@ -71,12 +71,16 @@ export class Node {
 Node.registerType('Scope');
 
 export function is(node, type) {
-    return (
-        node &&
-        (node.type === type ||
-            (IS_ALIAS_OF[type] && IS_ALIAS_OF[type][node.type]) ||
-            t.is(node, type))
-    );
+    if (node) {
+        if (node.type === type) {
+            return true;
+        }
+        if (IS_ALIAS_OF[type] && IS_ALIAS_OF[type][node.type] === true) {
```

Comment:
That is actually more readable yes, but `=== true` is not necessary

Reply:
Yes. I have added `=== true` for readability. Is it okay? Should I remove it?

Reply:
Well I would normally say its fine if it was not this project. You probably should.

Also the `=== true` part was not I meant good for readability actually. seperating the condition is better.